draft-ietf-privacypass-batched-tokens.md

```diff
@@ -414,14 +416,17 @@ corresponds to `nonce`, the i-th nonce that was sampled in
 
 ~~~
 struct {
-    uint16_t token_type;
+    uint16_t token_type; /* 0x0001 or 0x0005 */
     uint8_t nonce[32];
     uint8_t challenge_digest[32];
     uint8_t token_key_id[32];
     uint8_t authenticator[Nh];
 } Token;
 ~~~
 
+The constant `Nh` is as defined in {{OPRF, Section 4}} and denotes the output
```

Comment:
i had to cross check. Section 4 is the overall definition, and therefore fits both ristretto and p-384